pkg/shared/accountmgmtutil/ams.go

```diff
@@ -72,3 +73,25 @@ func GetOrganizationID(ctx context.Context, conn connection.Connection) (account
 
 	return account.Organization.GetId(), nil
 }
+
+// PickInstanceType - Standard instance always wins!!!
+// This function should not exist but it does represents some requirement
+// from business to only pick one instance type when two are presented.
+// When standard instance type is present in user instances it should always take precedence
+func PickInstanceType(amsType *[]string) (string, error) {
```

Comment:
@pb82 Because new API returns both standard and developer types 
client logic is really complex and cannot be made generic.

We will need to hardcode this logic and possibly break cli
later phase.

Do you think it is possible to check user ams quota types on backend and return only
instance sizes for the quota that user have access to

Reply:
Lets chat about it next week